app/helpers/markdown_helper.rb

```diff
@@ -0,0 +1,15 @@
+module MarkdownHelper
+  def parse(text)
+    markdown.render(text).html_safe
```

Comment:
__[RuboCop]__

Rails/OutputSafety: Tagging a string as html safe may be a security risk.


<!-- Comment by [SideCI](https://sideci.com) -->